website/docs/api/architectures.md

```diff
@@ -144,9 +144,9 @@ argument that connects to the shared `tok2vec` component in the pipeline.
 Construct an embedding layer that separately embeds a number of lexical
 attributes using hash embedding, concatenates the results, and passes it through
 a feed-forward subnetwork to build mixed representations. The features used are
-the `NORM`, `PREFIX`, `SUFFIX` and `SHAPE`, which can have varying definitions
-depending on the `Vocab` of the `Doc` object passed in. Vectors from pretrained
-static vectors can also be incorporated into the concatenated representation.
+the `NORM`, `PREFIX`, `SUFFIX` and `SHAPE`, and they are extracted with a
+[FeatureExtractor](#FeatureExtractor) layer. Vectors from pretrained static
```

Comment:
```suggestion
[FeatureExtractor](/api/architectures#FeatureExtractor) layer. Vectors from pretrained static
```

This is a nitpick but that little icon that's shown next to the architectures is picked based on the URL so we sometimes have to use a full link, even if it's on the same page so it looks nicer.